### PR TITLE
Added Sveltefire

### PR DESCRIPTION
**Pull request by lguzzon**
See you on SvelteSociety site ...

### CODE REVIEW

**Review by kevmodrome — DISMISSED** [review]
Please remove the firebase tag! 👍

**Comment by lguzzon** [comment]
> Please remove the firebase tag! 👍

Done!!! 👌

**Comment by stephane-vanraes** [comment]
it could use the tag `stores and state` though ?

and _maybe_ we could introduce a `cloudservices` tag ? what do you think @kevmodrome ?

**Comment by kevmodrome** [comment]
@stephane-vanraes sounds good! Completely missed this :(

File: src/pages/components/components.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "title": "sveltefire",
+    "url": "https://github.com/codediodeio/sveltefire",
+    "npm": "https://www.npmjs.com/package/sveltefire",
```
Comment (MacFJA) [inline src/pages/components/components.json]: This should be the NPM package name, not the URL to the NPM page
```suggestion
    "npm": "sveltefire",
```

**Review by MacFJA — DISMISSED** [review]

**Review by stephane-vanraes — APPROVED** [review]

**Review by stephane-vanraes — APPROVED** [review]

**Pull request merged**